Scripts/Core/EditorVR.cs

```diff
@@ -190,7 +190,7 @@ void Awake()
 			UnityBrandColorScheme.sessionGradient = UnityBrandColorScheme.GetRandomGradient();
 
 			var sceneObjectModule = AddModule<SceneObjectModule>();
-			sceneObjectModule.shouldPlaceObject = (obj, targetScale) =>
+			sceneObjectModule.tryPlaceObject = (obj, targetScale) =>
```

Comment:
I renamed it because if the object is inside the miniworld, we do actually place the object. That is pretty much all this delegate is responsible for. This was where I had to add code to add the object to the spatial hash.

I have reversed the logic as you described.